src/components/util/format.ts

```diff
@@ -41,10 +42,23 @@ const dateFormatter = new Intl.DateTimeFormat();
 export default function formatValue(str: string = '', opt: Type | Options = 'string') {
   if (str === null) return null;
 
-  const { type, dateFormat, meta, truncate, dps }: Options =
+  const { abbreviateLongNumbers, type, dateFormat, meta, truncate, dps }: Options =
     typeof opt === 'string' ? { type: opt } : opt;
 
-  if (type === 'number') return wrap(numberFormatter(dps).format(parseFloat(str)));
+  if (type === 'number' && !abbreviateLongNumbers)
+    return wrap(numberFormatter(dps).format(parseFloat(str)));
+
+  if (type === 'number' && abbreviateLongNumbers) {
+    const num = parseFloat(str);
+    if (isNaN(num)) return wrap(str);
+    return wrap(
+      num.toLocaleString('en-US', {
+        maximumFractionDigits: dps ?? 2,
+        notation: 'compact',
+        compactDisplay: 'short',
+      }),
+    );
+  }
```

Comment:
this is optional, but it simplifies a bit:

```suggestion
  if (type === 'number') {
     if(abbreviateLongNumbers) {
        const num = parseFloat(str);
         if (isNaN(num)) return wrap(str);
         return wrap(
           num.toLocaleString('en-US', {
             maximumFractionDigits: dps ?? 2,
             notation: 'compact',
             compactDisplay: 'short',
           }),
         );
     } else {
          return wrap(numberFormatter(dps).format(parseFloat(str)));  
     }
  }
```

Reply:
Nice, thanks!